Sources/StreamChatUI/ChatMessageList/ChatMessageListTitleView_Tests.swift

```diff
@@ -0,0 +1,65 @@
+//
+// Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+import StreamChat
+@testable import StreamChatUI
+import XCTest
+
+private typealias ChatMessageListTitleView = _ChatMessageListTitleView<NoExtraData>
+
+final class ChatMessageListTitleView_Tests: XCTestCase {
+    func test_emptyAppearance() {
+        let view = ChatMessageListTitleView().withoutAutoresizingMaskConstraints
+        NSLayoutConstraint.activate([
+            view.widthAnchor.pin(equalToConstant: 320),
+            view.heightAnchor.pin(equalToConstant: 44)
+        ])
+        AssertSnapshot(view)
+    }
+    
+    func test_defaultAppearance() {
+        let view = ChatMessageListTitleView().withoutAutoresizingMaskConstraints
+        view.content = ("Title", "Subtitle")
+        NSLayoutConstraint.activate([
+            view.widthAnchor.pin(equalToConstant: 320),
+            view.heightAnchor.pin(equalToConstant: 44)
+        ])
+        AssertSnapshot(view)
+    }
+    
+    func test_defaultAppearanceWithoutTitle() {
+        let view = ChatMessageListTitleView().withoutAutoresizingMaskConstraints
+        view.content = (nil, "Subtitle")
+        NSLayoutConstraint.activate([
+            view.widthAnchor.pin(equalToConstant: 320),
+            view.heightAnchor.pin(equalToConstant: 44)
+        ])
+        AssertSnapshot(view)
+    }
+    
+    func test_defaultAppearanceWithoutSubtitle() {
+        let view = ChatMessageListTitleView().withoutAutoresizingMaskConstraints
+        view.content = ("Title", nil)
+        NSLayoutConstraint.activate([
+            view.widthAnchor.pin(equalToConstant: 320),
+            view.heightAnchor.pin(equalToConstant: 44)
+        ])
+        AssertSnapshot(view)
+    }
+    
+    func test_customAppearance() {
```

Comment:
This should be probably called which appearance technique this test uses for custom appearance, i.e. `test_appearanceCustomization_usingAppearanceHook` which is used in other tests. Also looks like `UIConfig` changing tests are missing